implementations/elixir/ockam/ockly/checksum-Elixir.Ockly.Native.exs

```diff
@@ -0,0 +1,3 @@
+%{
+  "libockam_rust_elixir_nifs-v0.117.0-nif-2.15-aarch64-unknown-linux-gnu.so.tar.gz" => "sha256:f26c30f1a261ef8fba57c303eff08ea7cb55de1cc1467d5b3e38d782c1681b3a",
```

Comment:
This checksum was generated from the NIF created in my personal repository https://github.com/metaclips/ockam/releases/tag/ockam_v0.117.0